posix-configs/eagle/200qx/mainapp.config

```diff
@@ -1,9 +0,0 @@
-uorb start
```

Comment:
The removal was intentional. Those files were not being used anywhere. Px4.config is there as well and was being used instead for eagle.